image/Makefile

```diff
@@ -18,34 +29,30 @@ help: ## show make targets
 .PHONY: clean
 clean: ## remove build artifacts
 	-$(RM) $(ENGINE_DIR)/Dockerfile.engine
-	-docker rmi $(DOCKER_HUB_ORG)/$(ENGINE_IMAGE):$(STATIC_VERSION)
-	-rm -f image-linux
+	-$(RM) $(ENGINE_DIR)/Dockerfile.engine-dm
+	-docker rmi $(IMAGE_WITH_TAG)
+	-rm -f image-linux-*
 
 .PHONY: image
 image: image-linux
```

Comment:
Should this build both images so we make sure they're "tied at the hip" and always built together?  

Also related, I think there are other places in the CI pipeline where it builds "image-linux" which means it's not going to build this image the way this is structured, which seems potentially wrong.